content/community/community-team/engagement-channels/contents.lr

```diff
@@ -0,0 +1,173 @@
+
+**Address:** usability-announce@creativecommons.org
+
+### Community Team group
+
+This group is for all CC tech staff and everyone on the Community Team.
+
+**Address:** community-team@creativecommons.org
+
+## GitHub
+
+All our code is hosted on GitHub, we use issues to track bugs and identify
+features and pull requests to improve the software and drive it forward. All
+Community Team members with the role of
+[Project Collaborator](/community/community-team/project-roles/#project-collaborator)
+or higher are added to the
+[CC organisation](https://github.com/creativecommons/)
+on GitHub.
+
+We use GitHub teams to manage access levels for all members of the organisation.
+Each role in a particular project corresponds to a certain access level over the
+repositories that are a part of that project. As you change your role, you will
+automatically be added to the appropriate team.
+
+The [GitHub docs](https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/repository-permission-levels-for-an-organization#repository-access-for-each-permission-level)
+are the best place to learn about these access levels.
+
+### Read access
+
+Read access offers no special privileges over issues or pull requests. You can
+nevertheless contribute to the repository by forking it to your account and
+making PRs as usual. You can also partake in discussions and help new
+contributors get started when fixing simpler issues.
+
+### Triage access
+
+Triage access grants you the privileges required to review incoming issues and
+classify them by priority, goal, value added to the project and so on and apply
+the appropriate labels to them. You will be added to a project's `CODEOWNERS`
+and will automatically be assigned PRs to review. You can approve them or
+request changes as necessary and can prevent a PR from being merged.
+
+### Write access
+
+Write access grants you the privileges required to create branches on the
+repository, push to the repository directly and merge pull requests that have
+been approved. You can even commit changes to open pull requests to make them
+ready for merge and can draft new releases for the project.
+
+### Maintain access
+
+Maintain access grants you the privileges to manage almost all aspects of the
+repository, except any destructive action or sensitive information. You can
+modify the repositories' topics, description and social cards among other
+settings.
+
+## Twitter
+
+Twitter is our preferred social media platform. We don't have a LinkedIn or
+Facebook presence. We generally post updates from our projects, introduce newly
+published blog posts, announce new releases of our software and any other
+interested information related to both FOSS and Creative Commons.
+
+While [CC Open Source](https://twitter.com/cc_opensource) has a rather humble
+Twitter following, our tweets are often retweeted by
+[the parent Creative Commons handle](https://twitter.com/creativecommons).
+
+### Vote access
```

Comment:
I don't think we have a formal way of granting this access.

Reply:
Merged Vote, Suggest into Edit in bde9b54.